build/azdo/templates/setup.yml

```diff
@@ -11,5 +12,6 @@ steps:
     inputs:
       targetType: inline
       script: |
-        wget https://github.com/russellseymour/taskctl/releases/download/v${{ parameters.TaskctlVersion }}/taskctl_${{ parameters.TaskctlVersion }}_linux_amd64.tar.gz -O /tmp/taskctl.tar.gz
+        rm -rf /tmp/taskctl.tar.gz
```

Comment:
Is this simply to ensure that the tar.gz doesn't already exist so we don't run into conflicts if someone used multiple setup jobs in the same step?

Reply:
Basically when on self hosted agents (which happens occasionally) the next step fails if the zip already exists